modules/110-istio/docs/README.md

```diff
@@ -8,13 +8,10 @@ webIfaces:
 
 | Istio version | [K8S versions supported by Istio](https://istio.io/latest/docs/releases/supported-releases/#support-status-of-istio-releases) |          Status in D8          |
 |:-------------:|:-----------------------------------------------------------------------------------------------------------------------------:|:------------------------------:|
-|     1.12      |                                 1.19 &#8432;, 1.20 &#8432;, 1.21 &#8432;, 1.22 &#8432;                                        | Deprecated and will be deleted |
-|     1.13      |                                    1.20 &#8432;, 1.21 &#8432;, 1.22 &#8432;, 1.23 &#8432;                                     | Deprecated and will be deleted |
-|     1.16      |                                1.22 &#8432;, 1.23 &#8432;, 1.24 &#8432;, 1.25 &#8432; &#8432;                                 |           Deprecated           |
-|     1.19      |                    1.25 &#8432; &#8432;, 1.26 &#8432; &#8432;, 1.27 &#8432; &#8432;, 1.28 &#8432; &#8432;                     |           Supported            |
+|     1.16      |                                                1.22(!), 1.23(!), 1.24(!), 1.25                                                | Deprecated and will be deleted |
```

Comment:
@z9r5 The `&#8432;` stuff doesn't work. Please, suggest a nice approach to highlight the versions as non-supported.
<img width="721" alt="image" src="https://github.com/deckhouse/deckhouse/assets/18152319/5c5854c6-4c71-4251-8ad2-15e70cc50e4c">

Reply:
@tarabrind I've pushed the commit. Please check, does it look properly on macOS + Chrome?

Reply:
Yes, it works correctly now on macOS + Chrome